pcsx2/IopMem.cpp

```diff
@@ -490,3 +490,14 @@ void __fastcall iopMemWrite32(u32 mem, u32 value)
 		}
 	}
 }
+
+std::string iopMemReadString(u32 mem, int maxlen)
```

Comment:
In my humble opinion, you ought to compute `maxlen` based on mem and the IOP memory size. This way you could remove the while loop below with only a string constructor that will use the safe  `maxlen` limit.